mlir/test/Dialect/Linalg/invalid.mlir

```diff
@@ -725,6 +747,15 @@ func.func @broadcast_size_1_extension_not_supported(
       dimensions = [1]
   func.return %bcast : tensor<4x?x16xf32>
 }
+// -----
+
+func.func @broadcast_no_operands()
+    -> tensor<4x?x16xf32> {
+  linalg.broadcast dimensions = [1]
```

Comment:
Can you add another test case for the generic form? Same for the `linalg.transpose` and `linalg.map`.